Products/Zuul/facades/templatefacade.py

```diff
@@ -268,7 +268,7 @@ def getThresholdDetails(self, uid):
         """
         threshold = self._getObject(uid)
         template = threshold.rrdTemplate()
-        info = IThresholdInfo(threshold)
+        info = IInfo(threshold)
```

Comment:
Why? Was something incompatible with IThresholdInfo? Looking at IThresholdInfo, I think we'd want to keep using it.

You'll get an IThresholdInfo in most cases anyway, I just think we should error visibly rather than in the js silently.